test/FateRewardControllerV3.test.js

```diff
@@ -0,0 +1,234 @@
+const { ethers } = require("hardhat")
+const { expect } = require("chai")
+
+const { getBigNumber } = require("./utilities")
+const { advanceBlock, advanceBlockTo } = require("./utilities/time")
+const { BigNumber } = require('@ethersproject/bignumber')
+
+
+// yarn test test/FateRewardControllerV3.test.js
+describe("FateRewardControllerV3", () => {
+    const epoch_period_blocks = 30 * 60 * 24 * 7 * 8 // 8 weeks
+    const startBlock = 10
```

Comment:
You'll need to account for the epoch starting at week 14 (right after 13 ending)